packages/medusa-cli/src/util/token-store.ts

```diff
@@ -0,0 +1,12 @@
+const ConfigStore = require("configstore")
```

Comment:
todo: I believe we can remove this along with the `whoami` command

Reply:
do you mean I remove the entire files?

Reply:
The login can be removed as well right?